packages/number-input/src/Arrows/Arrows.types.ts

```diff
@@ -16,8 +16,14 @@ export interface ArrowsProps {
    */
   onKeyDown: (e: React.KeyboardEvent) => void;
 }
+export interface ArrowsProps extends BaseArrowProps {
+  /**
+   * Determines horizontal positioning of arrows
```

Comment:
can we just add size  to arrow props?

Reply:
`size` is used to position the group of arrows rather than positioning each one separately which I think is less code, so I prefer to keep the prop on `Arrows` rather than drill further to each singular`Arrow`